composer.json

```diff
@@ -15,6 +15,7 @@
     },
     "require-dev": {
         "behat/behat": "^3.4",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2",
```

Comment:
Why is this package suddenly needed?

Shouldn't the minimum version of `leviy/coding-standard` just be upped?